src/Manipulator/ObjectManipulator.php

```diff
@@ -76,6 +76,18 @@ private static function callGetter(object $object, string $fieldName): object
         $method = sprintf('get%s', $inflector->classify($fieldName));
 
         if (!(\is_callable([$object, $method]) && method_exists($object, $method))) {
+            if (property_exists($object, $fieldName)) {
+                $ref = new \ReflectionProperty($object, $fieldName);
+
+                if ($ref->isPublic()) {
+                    return $object->$fieldName;
+                }
+
+                throw new \RuntimeException(
+                    sprintf('Property %s::$%s does not accessible.', ClassUtils::getClass($object), $fieldName)
+                );
```

Comment:
You can remove this exception.

It will throw the next one:
```
throw new \RuntimeException(
     sprintf('Method %s::%s() does not exist.', ClassUtils::getClass($object), $method)
);
```

Reply:
Even better:
```
if (\is_callable([$object, $method]) && method_exists($object, $method)) {
   return $object->$method();
}

if (property_exists($object, $fieldName)) {
    $ref = new \ReflectionProperty($object, $fieldName);

    if ($ref->isPublic()) {
        return $object->$fieldName;
    }
}

/*
 * NEXT_MAJOR: Use BadMethodCallException instead
 */
throw new \RuntimeException(
    sprintf('Method %s::%s() does not exist.', ClassUtils::getClass($object), $method)
);
```